hebiros_advanced_examples/src/rviz/rosie_gazebo_model.rviz

```diff
@@ -0,0 +1,287 @@
+Panels:
```

Comment:
I would prefer to go ahead and hold off on adding this until we had something that referenced it as well.